eng/pipelines/scripts/Compare-CurrentToCodegeneration.ps1

```diff
@@ -0,0 +1,58 @@
+<#
+.SYNOPSIS
+Invokes all 'Update-Codegeneration.ps1' scripts found within the specified directory and compares it against current
+code.
+
+.DESCRIPTION
+Invokes all 'Update-Codegeneration.ps1' scripts found within the specified directory and compares it against current
+code.
+
+If the regenerated code is different than the current code this will tell the differences, the files the differences
+are in, and exit with a failure status.
+
+.PARAMETER Directory
+The directory that will be searched for 'Update-Codegeneration.ps1' scripts. The default is the root directory of the
+Azure SDK for Java repository. CI jobs should use the 'ServiceDirectory', such as /sdk/storage.
+#>
+
+param(
+  [Parameter(Mandatory = $false)]
+  [string]$Directory
+)
+
+$path = ""
+if ($Directory) {
+  $path = $Directory
+}
+
+Write-Host "
```

Comment:
Does this work? I would expect if you wanted a multi-line string it would need to look something like:
```
Write-Host @"

content

"@
```

Reply:
It does seem to work like this, not sure why or what is missing from my knowledge of this.

Reply:
Hmmmmm.... I do not understand what they changed to support this either.